.github/workflows/ci.yml

```
@@ -0,0 +1,60 @@
+---
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+name: CI
+jobs:
+  ci:
+    name: CI
+    runs-on: [self-hosted, heavy]
+    steps:
+      - name: Install rust
+        run: |
+          rustup show
+          rustup default ${{ env.RUST_VERSION }}
+
+      - name: Clone the repository
+        uses: actions/checkout@v3
+
+      - name: Restore cache
+        run: |
+          cache-util restore cargo_git cargo_registry sandbox:${{ env.NEAR_SANDBOX_BIN_PATH }}
+
+      - name: Check fmt
+        run: cargo fmt -- --check
+      - name: Lint checker
+        run: cargo clippy --all-features -- -D warnings
```

```suggestion
        run: cargo clippy --all-features --all -- -D warnings
```
The parameter `--all` checks tests as well.